code/src/components/StartSurvey.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export const StartSurvey = ({ section }) => {
+  return (
+    // eslint-disable-next-line react/jsx-no-useless-fragment
+    <>
+      {section === 0 && (
```

Comment:
So, as I understand now, you have if conditional in some component instead of wrapping it all in App.js. Cool. I think if it works, it works! And this shows how much we can do in React. There are many ways of making it works 👍🏼